store/entity-store.js

```diff
@@ -161,14 +182,20 @@ window.D2L.Siren.EntityStore = {
 			if (!entity || bypassCache) {
 				const headers = new Headers();
 
-				tokenValue && headers.set('Authorization', 'Bearer ' + tokenValue);
+				const shouldAttachToken = this._shouldAttachToken(sirenLinkOrHref);
+
+				shouldAttachToken && tokenValue && headers.set('Authorization', 'Bearer ' + tokenValue);
```

Comment:
`shouldAttachToken &&` not strictly necessary since we make `tokenValue` empty as well, but to be sure in case something changes.